### PR TITLE
INVGOV-14001 /job_roles/merge Node.js API endpoint

### PR DESCRIPTION
**Pull request by klauselhai**
https://io1dev.atlassian.net/browse/INVGOV-14001

https://dev.invotra.com/?url=https://raw.githubusercontent.com/invotra/api/INVGOV-14001.SWAGGER-5.23/api.swagger.yaml

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -7263,6 +7263,36 @@ paths:
         - SessionCookieHeader: []
         - APIKeyHeader: []
         - CognitoJwt: [ invotra/job_roles:write ]
+  '/job_roles/merge':
```
Comment (sam-ludlow) [inline api.swagger.yaml:7266]: I think we should do:

PUT ​/job_roles​/{job_role_id}/merge
body { merge_uuids[] }

File: api.swagger.yaml [hunk 2]
```diff
@@ -7263,6 +7263,36 @@ paths:
...
+                $ref: '#/components/schemas/successful-merge-job-roles-with-count-schema'
+        '400':
+          description: Job roles were not merged due to malformed JSON.
+        '403':
+          description: 'User is not authorized to merge job roles.'
+        '422':
+          description: |
+            The job roles were not merged due to Drupal validation failure or JSON
+            schema validation failure.
+      description: Provision a new job role.
```
Comment (sam-ludlow) [inline api.swagger.yaml:7285]: Bad description

**Review by sam-ludlow — CHANGES_REQUESTED** [review]

**Review by sam-ludlow — APPROVED** [review]
Looks good

**Comment by MireilleHelle** [comment]
Closing task as we’re now focusing on building only API gaps for which Invotra Consulting have a direct need.

**Pull request closed without merging**